### ISSUE TITLE
Timely Intervention

### ISSUE THREAD

**Issue by katenamgayoung** [IITIDIDX597/F2023_Team2#31]
**Goal Statement**
For: _Incarcerated Mothers, Children and Social Workers, Prison Staff_
Who: _female prisoners who are mothers and are separated from their children cannot be a consistent influence on their children due to incarceration and cannot be a positive influence on them_
Your: _Hedwig, a tablet-based application_
Is a: _provides ways for incarcerated mothers to communicate and for their children to share through pictures, text and video._
That: _which reconnects mothers and children and to maintain their relationship to reduce incidents based on isolation within correctional facilities as well as integrate security and regulatory compliance._

**Hill Statement**
Who: _Incarcerated Mothers and Children, Social Workers and Prison staff_ 
What: _Effectively monitors prisoner and child interactions to understand positive experiences that could affect prisoner behavior as well as negative behaviors that may harm the mother and child._
Wow: _through an integrated platform where content can be monitored, reducing incidents of technology misuse 24 hours a day_

**Sub-Hill Statement**
Information shared by incarcerated mothers and children will be notified to social worker in real-time

**Story Statement**
As a: social worker,
I want: to be notified in real-time about the information shared by incarcerated mothers and children using the Hedwig platform
So that: I can provide timely support and intervention.

**Comment by akallish** [comment]
@katenamgayoung, @bhavanisp, @crutuja Good iteration 0. Consider this iteration and refine :

As a: social worker
Comment : Good and specific
I want: to be notified in real-time about the information shared by incarcerated mothers and children using the Hedwig platform
Comment : Consider based on your sub-hill and hill statements "to be notified about all posts but highlight ones that have been blocked/flagged by Hedwig which could be considered dangerous for either party
So that: I can provide timely support and intervention.
Comment : Consider "so social worker can provide timely intervention and where appropriate consulting with prison officials."

**Comment by crutuja** [comment]
As a social worker, I want to receive notifications for all posts but with a specific emphasis on those flagged or blocked by Hedwig, which may be deemed potentially harmful to either party. So that the social workers can ensure promptly intervene and, when necessary, engage in consultations with prison officials.

**Comment by crutuja** [comment]
Story statement version- 3
As a social worker, I want to stay informed about all posts, with a particular focus on those flagged or blocked by Hedwig due to their potential harm to either party. So that the social workers can facilitate swift intervention, and when deemed necessary, enable consultations with prison officials for a more effective response.

**Comment by crutuja** [comment]
Hi @katenamgayoung i have added the issue to done please review it.

**Comment by katenamgayoung** [comment]
@crutuja I've reviewed it and could you move to the next step writing acceptance criteria with story version 3? Thank you!

**Comment by crutuja** [comment]
Acceptance criteria version 1

Given- I am a social worker with access to the monitoring system
When- I log in and access the system dashboard
Then- the system displays a feed of all posts, placing a specific focus on those flagged or blocked by Hedwig due to potential harm. This allows me to stay informed, facilitating prompt intervention. Additionally, when needed, I can initiate consultations with prison officials for a more effective response to ensure the well-being of the involved parties.

**Comment by akallish** [comment]
@crutuja This is a good iteration 0 for acceptance criteria. I do think the "when" statement should have "the system displays a feed of all posts, placing a specific focus on those flagged or blocked by Hedwig due to potential harm. This allows me to stay informed, facilitating prompt intervention." The "then" statement might be a course of action = "When needed, I can initiate consultations with prison officials for a more effective response to ensure the well-being of the involved parties."

What do you think?

**Comment by crutuja** [comment]
Ya i agree with you, will improvise in the next iteration, thank-you!  @akallish

**Comment by crutuja** [comment]
Acceptance criteria version 2

Given- I am a social worker with access to the monitoring system.
When- I log in and access the system dashboard, where the system displays a feed of all posts. It places a specific focus on those flagged or blocked by Hedwig due to potential harm. This allows me to stay informed, facilitating prompt intervention.
Then- I can initiate consultations with prison officials for a more effective response to ensure the well-being of the involved parties.

**Comment by akallish** [comment]
@crutuja I think more of the then should be in the when. When the dashboard shows X; then I will do action

**Comment by crutuja** [comment]
yes i agree, sorry i did some mistake while editing, but have updated the previous comment.
